src/browser/modules/D3Visualization/components/OverviewPane.tsx

```diff
@@ -66,120 +57,63 @@ function OverviewPane({
   hasTruncatedFields,
   nodeCount,
   relationshipCount,
-  selectLabel,
-  selectRelType,
-  stats,
-  legendItem
+  stats
 }: OverviewPaneProps): JSX.Element {
   const { relTypes, labels } = stats
 
   return (
     <>
-      <PaneHeader>
-        {hasTruncatedFields && (
-          <StyledTruncatedMessage>
-            <Icon name="warning sign" /> Record fields have been
-            truncated.&nbsp;
-          </StyledTruncatedMessage>
+      <PaneHeader>Overview</PaneHeader>
+      <PaneBody maxHeight={frameHeight - 45}>
+        {labels && Object.keys(labels).length !== 0 && (
+          <>
+            Node labels
+            <StyledLegendInlineList>
+              {Object.keys(labels).map((label: string) => (
```

Comment:
I cut this feature due to time constraints, but thinking about the performance implications, and the fact that it could be fairly quick to implement now that the rest of the feature is in place I'll give it a second look.